### PR TITLE
Gstreamer plugin README edits

### PR DESCRIPTION
**Pull request by asajeffrey**
Updated docs to the gstreamer plugin, now with a recipe for streaming to youtube live!

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes do not require tests because we don't test READMEs

### CODE REVIEW

File: ports/gstplugin/README.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ GST_PLUGIN_PATH=target/gstplugins \
...
+  gst-launch-1.0 -e servowebsrc \
+    ! video/x-raw\(memory:GLMemory\),framerate=50/1,width=1960,height=1080 \
+    ! glvideoflip video-direction=vert \
+    ! glcolorconvert \
+    ! gldownload \
+    ! x264enc bitrate=6000 \
+    ! flvmux name=mux \
+    ! rtmpsink location="rtmp://a.rtmp.youtube.com/live2/x/$TOKEN"
+    audiotestsrc wave=silence \
+    ! voaacenc bitrate=128000
```
Comment (jdm) [inline ports/gstplugin/README.md:61]: There are a couple \s missing on these lines.

Comment (asajeffrey) [reply ports/gstplugin/README.md]: Fixed.

**Comment by CYBAI** [comment]
@bors-servo retry

- #25513

**Pull request merged**